### PR TITLE
5 create todo list markup

### LINKED ISSUE #5: Create a todo list markup
## What

Create a todo list markup.

## Deliveries

Single index.html file

### PR DESCRIPTION
**Pull request by chrislicodes**
## What

Created todo list markup

## Deliveries

index.html
sprites.svg

Closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,400&display=swap"
+        rel="stylesheet">
+
+    <meta name="theme-color" content="#fafafa">
+
+    <link rel="stylesheet" href="./css/style.css">
```
Comment (artdaw) [inline index.html:19]: You don't have styles so far, so please remove it/

File: index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    <script defer src="./js/app.js"></script>
```
Comment (artdaw) [inline index.html:20]: You also don't have JS, so please remove it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            <span class="heading__secondary">Getting things done - one line at a time</span>
+        </h1>
+    </header>
+    <main class="todo-container">
+        <div class="todo-app">
+            <div class="todo-app__header">
+                <form class="input-form" name="input-form" enctype="multipart/form-data">
+                    <div class="input-text-container">
+                        <input type="text" name="description" class="input-text"
+                            placeholder="The next thing I have to do is .." required>
```
Comment (artdaw) [inline index.html:36]:
```suggestion
                            placeholder="The next thing I have to do is .." required="required">
```

or

```suggestion
                            placeholder="The next thing I have to do is .." required="true">
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                    </div>
+                    <div class="input-control-container">
+                        <div class="input-time-info">
+                            <input type="date" name="date" max="2099-12-31" class="input-date" required>
```
Comment (artdaw) [inline index.html:40]:
```suggestion
                            <input type="date" name="date" max="2099-12-31" class="input-date" required="required">
```

or 

```suggestion
                            <input type="date" name="date" max="2099-12-31" class="input-date" required="true">
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+                            <input type="time" name="time" value="13:30" class="input-time" required>
```
Comment (artdaw) [inline index.html:41]:
```suggestion
                            <input type="time" name="time" value="13:30" class="input-time" required="required">
```

or

```suggestion
                            <input type="time" name="time" value="13:30" class="input-time" required="true">
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+                            <select name="category" id="category" class="input-category">
+                                <option value="coding">Coding</option>
+                                <option value="finance">Finances</option>
+                                <option value="household">Household</option>
+                            </select>
+                        </div>
+                        <div class="input-btn-container">
+                            <button class="input-btn btn">Add Task :&#41;</button>
```
Comment (artdaw) [inline index.html:49]: It is better to set `type` for button explicitly. It is `type="submit"` by default.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+                            </button>
+                            <button class="task__delete-btn" data-tag="delete">
+                                <svg>
+                                    <use xlink:href="./img/sprites.svg#icon-cross"></use>
+                                </svg>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (artdaw) [inline index.html:107]: If this is a list of 2 tasks it is not semantically right. 
You MUST use `<ul>` or `<ol>` to define lists in HTML.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+                    <div class="task-status-btns">
+                        <p>Status:</p>
+                        <input type="checkbox" id="task-open">
+                        <label for="task-open" class="btn">Open</label>
+                        <input type="checkbox" id="task-done">
+                        <label for="task-done" class="btn">Done</label>
+                    </div>
+                </form>
+            </div>
+        </div>
```
Comment (artdaw) [inline index.html:128]: I don't remember we have a filtering functionality, so I guess this MUST be removed.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+    </main>
+</body>
+
+</html>
```
Comment (artdaw) [inline index.html:132]: Please, add empty line at the end of the file.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+    <link rel="stylesheet" href="./css/style.css">
+    <script defer src="./js/app.js"></script>
+</head>
+
+<body>
+    <header class="heading-container">
+        <h1 class="heading">
+            <span class="heading__primary">Minimal ToDo - List</span>
+            <span class="heading__secondary">Getting things done - one line at a time</span>
+        </h1>
```
Comment (artdaw) [inline index.html:28]: 1. This is not semantically right. You have a choice of different headers from `h1` to `h6`, so h1 might be a primary header and h2 might be used as secondary header/
2. No need for spans.

**Review by artdaw — CHANGES_REQUESTED** [review]
## Overall comments

1. Remove all classes attributes since we don't apply any style yet.
2. Remove all <div> elements since they are not needed for this task, they have no semantic. They might be used in future once we style our list of todos.
3. Please, address all single comments

File: index.html [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+                <button type="submit" data-tag="complete">
+                    <svg>
+                        <use xlink:href="./img/sprites.svg#icon-checkmark"></use>
+                    </svg>
+                </button>
+                <button type="submit" data-tag="delete">
+                    <svg>
+                        <use xlink:href="./img/sprites.svg#icon-cross"></use>
+                    </svg>
+                </button>
```
Comment (artdaw) [inline index.html:55]: ## Suggestion

I would think about using [`<article>`](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/article) element to group description, date and actions into 1 widget.

Comment (chrislicodes) [reply index.html]: I added the article tag as discussed, also each article element requires an Heading Tag, so I replaced the p tag.

**Review by artdaw — APPROVED** [review]

**Pull request merged**